models/art.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+// Each Movement needs to have the following features:
+// - A name (required).
+// - A description (no longer than 200 characters).
+// - Starting and ending years (as strings) for the time when the movement
+//    was most prominent.
+// - An 'ancestor', another art movement that this movement descends from.
+
+const mongoose = require('mongoose');
+
+const artSchema = new mongoose.Schema({
+  name: {
+    type:String,
+    required:true
+  },
+  description: String,
+  startingYear: String,
+  endingYear: String,
+  _owner: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: 'User',
+  },
```

Comment:
Should be `_ancestor` and refer to another Movement (or, as you've named it Art).